kratos/testing/distributed_test_case.cpp

```diff
@@ -59,7 +59,7 @@ std::string DistributedTestCase::Info() const
 void DistributedTestCase::CheckRemoteFailure()
 {
     bool success_on_this_rank = GetResult().IsSucceed();
-    const DataCommunicator& r_comm = DataCommunicator::GetDefault();
+    const DataCommunicator& r_comm = ParallelEnvironment::GetDefaultDataCommunicator();
```

Comment:
Why not using the one from `testing` namespace in here?

Reply:
(would be the same but seems more consistent)

Reply:
I would like to use the `Testing::GetDefaultDataCommunicator()` exclusively in the tests
This is not a test, hence I would leave it as is, as it is IMO one of the few valid usecases of `ParallelEnvironment::GetDefaultDataCommunicator`

If you insist I can change it but I would prefer to leave it as is

Reply:
No no, it was just a suggestion. Approving.